driver/ppm_events.c

```diff
@@ -204,10 +204,6 @@ inline int sock_getname(struct socket* sock, struct sockaddr* sock_address, int
 
 		sin->sin_family = AF_INET;
 		if (peer) {
-			if (!inet->inet_dport ||
```

Comment:
Apart from the fact that I don't understand why we return an `errno` here since the caller expects a "size", why do we need to remove these lines in this patch? also please note that we have the specular code for ipv6

Reply:
> why do we need to remove these lines in this patch?

Removing that check makes `sock_getname` to work with non-TCP sockets.
Also `bpf_sockgetname` doesn't have this check.
> also please note that we have the specular code for ipv6

Oops.. I didn't see that